src/Control/Monad/Trans/Either.hs

```diff
@@ -127,3 +133,90 @@ hoistEitherT :: (forall b. m b -> n b) -> EitherT x m a -> EitherT x n a
 hoistEitherT f =
   EitherT . f . runEitherT
 {-# INLINE hoistEitherT #-}
+
+------------------------------------------------------------------------
+-- Error handling
+
+-- | Handle an error. Equivalent to 'catchError' in 'mtl'.
+handleEitherT :: Monad m => EitherT e m a -> (e -> EitherT e m a) -> EitherT e m a
+handleEitherT thing handler = do
```

Comment:
This has the same name and similar functionality to my PR #4 .

Reply:
Yeah - we have the same terminology problem. EitherT is useful without an exception-throwing base monad, and `either`-the-package used to ship the pure versions only.

It is probably good to have both pure and exception-handling versions of these functions available?

Reply:
I'd prefer to avoid primes on the names if possible.

Maybe my versions should be called some variant on `handleExceptEitherT` and then your versions can simply be `handleEitherT`.

Reply:
Your use of the name `handleEitherT` is the only thing that clashes with my PR. What do you think of renaming yours to `handleLeftT` ?